libraries/AP_Baro/AP_Baro_BMP280.h

```diff
@@ -34,6 +34,12 @@ class AP_Baro_BMP280 : public AP_Baro_Backend
     int32_t _t_fine;
     float _pressure;
     float _temperature;
+    float _mean_temperature;
+    uint8_t _mask;
+    bool _letterY;
```

Comment:
Some comments are welcome. It is totally unclear that this means.

Reply:
names of variables are self explaining

Reply:
see PR's description